change/@microsoft-teams-js-cc1d35f9-c5f8-4d2b-9161-2d3c53d6d593.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added new `appNotification` capability for apps to send notifications to the user through the host",
```

Comment:
This change is not introducing app-Notification in teams-js. It is fixing the typos and bugs